src/simplify_algebra.cpp

```diff
@@ -851,6 +853,92 @@ struct find_div_const
     }
 };
 
+struct find_zero_div_const
+{
+    auto matcher() const { return match::name("div")(match::arg(1)(match::has_value(0.0f))); }
+
+    void apply [[noreturn]] (const module& m, const match::matcher_result& r) const
+    {
+        if(enabled(MIGRAPHX_TRACE_PASSES{}))
```

Comment:
Can you check if `MIGRAPHX_TRACE_MATCHES` dumps the same information? I think it is, in that case we don't need another ENV_VAR

Reply:
Ah that makes sense. Ill try that.

Reply:
Still doesn't find it when I compile in here. I though we would have some param to gate off of for matchers no?